eventuate-util-spring-swagger/src/main/java/io/eventuate/util/spring/swagger/CommonSwaggerAutoConfiguration.java

```diff
@@ -11,7 +12,8 @@
 
 @Configuration
 @EnableSwagger2
-public class CommonSwaggerConfiguration {
+@ConditionalOnBean(EventuateSwaggerConfig.class)
+public class CommonSwaggerAutoConfiguration {
```

Comment:
There should be two classes: CommonSwaggerAutoConfiguration and CommonSwaggerConfiguration.
CommonSwaggerAutoConfiguration has the `@ConditionalOnBean(EventuateSwaggerConfig.class)` and `@Imports`  `CommonSwaggerConfiguration`, which is unchanged.

Reply:
@cer Fixed